charts/redpanda/templates/_helpers.tpl

```diff
@@ -639,8 +639,10 @@ return licenseSecretRef.key checks deprecated values entry if current values emp
 - name: redpanda-{{ $name }}-cert
   mountPath: {{ printf "/etc/tls/certs/%s" $name }}
     {{- end }}
+    {{- if .Values.listeners.kafka.tls.requireClientAuth }}
 - name: mtls-client
-  mountPath: /etc/tls/certs/{{ template "redpanda.fullname" $ }}-client
+  mountPath: /etc/ls/certs/{{ template "redpanda.fullname" $ }}-client
```

Comment:
/ls/certs? should this not be tls? If you are correct here we should fix this, that doesnt make sense.